taskcluster/ci/mark-as-shipped/kind.yml

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: fenix_taskgraph.loader.multi_dep:loader
+
+transforms:
+    - fenix_taskgraph.transforms.multi_dep:transforms
+    - fenix_taskgraph.transforms.mark_as_shipped:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - push-apk
+
+primary-dependency: push-apk
+
+group-by: build-type
+
+only-for-build-types:
+    - beta
+    - production
+
+job-template:
+    description: Mark Fenix as shipped in ship-it
+    worker-type: ship-it
+    scopes:
```

Comment:
Question: Do we want to use a staging/production scope like gecko does? https://searchfox.org/mozilla-central/source/taskcluster/ci/release-mark-as-shipped/kind.yml#26-33

Reply:
Probably - I just wasn't sure of any `by-*` that would work here to achieve that.

Reply:
Got it! I agree `by-release-level` won't work since it's gecko-specific. `by-level` would, I think. That said, we can add this scope later.